lib/canvas-motion.js

```diff
@@ -1,13 +1,15 @@
-function CanvasMotion(canvasWidth, canvasHeight) {
+function CanvasMotion(canvasWidth, canvasHeight, p5Amplitude) {
   this.canvasWidth = canvasWidth;
   this.canvasHeight = canvasHeight;
 }
 
 CanvasMotion.prototype = {
-  moveFish: function(fishies, round) {
+  moveFish: function(fishies, round, p5Amplitude) {
     var self = this;
     fishies.forEach(function(fish) {
-      moveFishX(fish, round);
+      // console.log("moveFish " + p5Amplitude);
+      moveFishX(fish, round, p5Amplitude);
+      moveFishY(fish, p5Amplitude);
```

Comment:
creates method adjusting Y, only adds, thus fish move down only. We could experiment with subtracting the amplitude from a medium value and having them bounce up and down, too.